lib/sprockets/commoner/processor.rb

```diff
@@ -70,12 +67,13 @@ def call(input)
 
         babel_config = babelrc_data(filename)
 
-        result = input[:cache].fetch([filename, @cache_key, input[:data], babel_config]) do
-          transform(input[:data], options(input), paths: @env.paths)
-        end
+        result = transform(input[:data], options(input), paths: @env.paths)
 
         if result['metadata'].has_key?('required')
           result['metadata']['required'].each do |r|
+            unless r.end_with?('.coffee') || should_process?(r)
```

Comment:
Is there a better way to check for CoffeeScript files? I guess if it's a `.coffee.erb` the `.erb` is resolved and removed before getting here?